test/e2e/test_cluster_configs/flatcar/OWNERS

```diff
@@ -0,0 +1,2 @@
+maintainers:
+- invidian

```

Comment:
@invidian is there anyone else at kinvolk that would like to maintain this going forward?

Reply:
I'll ask around, perhaps it would be good to have more than one person around. Maybe @iaguis?

Reply:
Thanks for volunteering me :D

Happy to be added here.